Description: add sunway support
Author: Miao Changwei <miaochangwei@uniontech.com>

---
Last-Update: 2022-05-19

--- gdal-3.3.3+dfsg.orig/frmts/pcraster/libcsf/csftypes.h
+++ gdal-3.3.3+dfsg/frmts/pcraster/libcsf/csftypes.h
@@ -161,6 +161,11 @@ typedef double              REAL8; /* IE
 #  define CPU_BIG_ENDIAN
 #endif
 
+#ifdef __sw_64__
+/* linux/gcc defines this on sw_64 platform */
+#  define CPU_LITTLE_ENDIAN
+#endif
+
 /* endif probing */
 # endif
 
